generated/nidigital/nidigital/_attributes.py

```diff
@@ -100,6 +101,38 @@ def __set__(self, session, value):
         return self._underlying_attribute.__set__(session, value.value)
 
 
+class AttributeEnumWithConverter(AttributeEnum):
```

Comment:
wondering if we should make the generator smarter so that only the Attribute classes that are used get generated.... it would of course be a separate PR

Reply:
I have filed an issue for this #1762.